src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +19,16 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers
+                .stream()
+                .map(s -> s.split(","))
+                .flatMap(Arrays::stream)
```

Comment:
```suggestion
                .flatMap(s -> Arrays.stream(s.split(",")))
```
prefer 1 map (or flatMap) over several